src/screens/flows/WalletConnect/screens/Flows/components/RequestTransferHeader/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { StyleSheet } from 'react-native';
+
+const opacityWhite = '#ffffff16';
```

Comment:
why not Colors.White and the `opacity` prop?

Reply:
Because the opacity impacts the hole view and not just the color, this was the best way to make it work